<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateUQModelDialog</class>
 <widget class="QDialog" name="updateUQModelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Ensemble - Model Selection</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="nodeRadioButton">
     <property name="toolTip">
      <string>Choose this option if your simulation model
is a node in the flowsheet.</string>
     </property>
     <property name="text">
      <string>Use flowsheet</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="emulatorRadioButton">
     <property name="toolTip">
      <string>Choose this option if you want to train a response
surface that emulates your actual simulation model.
You will need training data saved as a PSUADE file.
If you'd like to change the response surface type and/or
Legendre order, edit these options directly in the file.</string>
     </property>
     <property name="text">
      <string>Use emulator (Response Surface)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="dataFileLabel">
         <property name="text">
          <string>Data File:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dataFileEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browseButton">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="outputLabel">
         <property name="text">
          <string>Select Output(s) of Interest:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="outputList"/>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="fileStatsLabel">
       <property name="text">
        <string>Response Surface Type:
Legendre Order:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>updateUQModelDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>updateUQModelDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
